Core ignored options: conn-ssl-trust-all

```
support-external-java-clients(M?)+32-10 $ java -jar ./systemtests/client_executable/cli-a rtemis-jms.jar sender --conn-ssl-verify-host false --conn-username test --count 10 --broker tcp://[broker]:61616 -- conn-ssl-trust-all true --address selector-queue --conn-password test --log-msgs json
ignored options: conn-ssl-trust-all
09:25:03,419 ERROR AMQ214016: Failed to create netty connection
javax.net.ssl.SSLHandshakeException: General SSLEngine problem
        at sun.security.ssl.Handshaker.checkThrown(Handshaker.java:1478)
        at sun.security.ssl.SSLEngineImpl.checkTaskThrown(SSLEngineImpl.java:535)
        at sun.security.ssl.SSLEngineImpl.readNetRecord(SSLEngineImpl.java:813)
        at sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:781)
        at javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:624)
        at io.netty.handler.ssl.SslHandler$SslEngineType$3.unwrap(SslHandler.java:281)
        at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1215)
        at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1127)
        at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1162)
        at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
        at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
        at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
        at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
        at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:797)
        at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:404)
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:304)
        at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: General SSLEngine problem
        at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
        at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1728)
        at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:304)
        at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
        at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
        at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
        at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
        at sun.security.ssl.Handshaker$1.run(Handshaker.java:966)
        at sun.security.ssl.Handshaker$1.run(Handshaker.java:963)
        at java.security.AccessController.doPrivileged(Native Method)
        at sun.security.ssl.Handshaker$DelegatedTask.run(Handshaker.java:1416)
        at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1364)
        at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1272)
        ... 17 more
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
        at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
        at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
        at sun.security.validator.Validator.validate(Validator.java:260)
        at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
        at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:281)
        at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:136)
        at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1501)
        ... 25 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
        at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
        at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
        at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
        at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
        ... 31 more
09:25:03,447 ERROR Failed to create session factory
javax.jms.JMSException: Failed to create session factory
        at org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory.createConnectionInternal(ActiveMQConnectionFactory.java:768)
        at org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory.createConnection(ActiveMQConnectionFactory.java:267)
        at com.redhat.mqe.acc.AccConnectionManager.<init>(AccConnectionManager.java:71)
        at com.redhat.mqe.acc.AccConnectionManagerFactory.make(AccConnectionManagerFactory.java:30)
        at com.redhat.mqe.acc.AccConnectionManagerFactory.make(AccConnectionManagerFactory.java:25)
        at com.redhat.mqe.lib.CoreClient.createConnection(CoreClient.java:87)
        at com.redhat.mqe.lib.SenderClient.startClient(SenderClient.java:73)
        at com.redhat.mqe.lib.Main.main(Main.java:46)
        at com.redhat.mqe.acc.Main.main(Main.java:27)
Caused by: ActiveMQNotConnectedException[errorType=NOT_CONNECTED message=AMQ119007: Cannot connect to server(s). Tried with all available servers.]
        at org.apache.activemq.artemis.core.client.impl.ServerLocatorImpl.createSessionFactory(ServerLocatorImpl.java:790)
        at org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory.createConnectionInternal(ActiveMQConnectionFactory.java:766)
        ... 8 more
```

--- a/cli-artemis-jms/src/main/java/com/redhat/mqe/acc/AccClientOptionManager.java
+++ b/cli-artemis-jms/src/main/java/com/redhat/mqe/acc/AccClientOptionManager.java
@@ -94,6 +94,7 @@ class AccClientOptionManager extends ClientOptionManager {
         CONNECTION_TRANSLATION_MAP.put(ClientOptions.CONN_TIGHT_ENCODING_ENA, "");
 
         // activemq-artemis/examples/features/standard/ssl-enabled-dual-authentication/src/main/resources/jndi.properties
+        // org.apache.activemq.artemis.core.remoting.impl.netty.TransportConstants for more
         CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_KEYSTORE_LOC, "keyStorePath");
         CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_KEYSTORE_PASS, "keyStorePassword");
         CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_TRUSTSTORE_LOC, "trustStorePath");
@@ -104,7 +105,7 @@ class AccClientOptionManager extends ClientOptionManager {
         CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_DIS_CIPHERED, "");
         CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_ENA_PROTOS, "enabledProtocols");
         CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_DIS_PROTOS, "");
-        CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_TRUST_ALL, "");
+        CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_TRUST_ALL, "trustAll");
         CTL_SSL_OPTIONS.put(ClientOptions.CON_SSL_VERIFY_HOST, "verifyHost");
         CONNECTION_TRANSLATION_MAP.putAll(CTL_SSL_OPTIONS);
     }

